WebRoot/js/zimbraMail/calendar/controller/ZmCalViewController.js

```diff
@@ -2233,17 +2233,17 @@ function(appt, mode) {
 			choiceLabel2 : ZmMsg.notifyOrganizer
 		});
 	}
-    if(this._deleteMode != mode){
-        var msg = ZmMsg.confirmCancelAppt;
-        if(appt.isRecurring()){
-            msg = (mode == ZmCalItem.MODE_DELETE_INSTANCE)
-    			? AjxMessageFormat.format(ZmMsg.confirmCancelApptInst, AjxStringUtil.htmlEncode(appt.name))
-    			: ZmMsg.confirmCancelApptSeries;
-        }
-        var msgDiv = document.getElementById(this._deleteNotifyDialog._confirmMessageDivId);
-        msgDiv.innerHTML = msg;
-        this._deleteMode = mode;
-    }
+
```

Comment:
looks like we have removed some code from https://github.com/Zimbra/zm-web-client/commit/341bdbaf103aacc7a601f6e0beaa7697dd6e4683
Please reverify https://bugzilla.zimbra.com/show_bug.cgi?id=73699

Reply:
I did verify this scenario and its working as expected.